### PR TITLE
:mortar_board: Topic overview --- Typical settings and behaviour 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Discuss typical (whatever that means) settings and typical (whatever that means) behaviours of an EA. 

### Testing
:+1: built local, looks good. 

### Additional Notes
Figures are from book. Book is used for the course. Book and slides are freely available online. Should be OK, but if there's a problem in the future, I'll change them out.

### CODE REVIEW

File: site/topics/overview/overview.rst [hunk 1]
```diff
@@ -315,9 +315,88 @@ Examples
...
+
+
+    * Population size
+
+        * Could be a few dozen or in the thousands
+
+
+    * Crossover rate (usually around 80%)
+
+        * Usually around 80%
```
Comment (twentylemon) [inline site/topics/overview/overview.rst:339]: usually around 80%  
usually around 80%

File: site/topics/overview/overview.rst [hunk 2]
```diff
@@ -315,9 +315,88 @@ Examples
...
+
+
+    * Mutation rate (usually around 10%)
+
+        * Usually around 10%
```
Comment (twentylemon) [inline site/topics/overview/overview.rst:344]: usually around 10%  
usually around 10%

**Review by twentylemon — APPROVED** [review]

File: site/topics/overview/overview.rst (hunk 2 above)
Comment (jameshughes89) [reply site/topics/overview/overview.rst:344]: image

Comment (jameshughes89) [reply site/topics/overview/overview.rst:344]: that was supposed to be a gif, but whoopsie...

**Pull request merged**